### ISSUE TITLE
Crashes upon finishing any race

### ISSUE THREAD

**Issue by DannyTheSniper** [Fireboyd78/mm2hook#94]
Hi there. Recently I've been experiencing crashes when finishing any race (both vanilla and mods). I have no other mods installed. As far as I remember, it might have been caused by some red lights or something.

Here's the log for reference:
[mm2.log](https://github.com/Fireboyd78/mm2hook/files/6812881/mm2.log)
Any help would be appreciated!

**Comment by david114** [comment]
Same Issue here, have you found a solution?

**Comment by DannyTheSniper** [comment]
Yes, I tried downgrading to an older version of MM2Hook that MysteryDriverX was using provided to me in the Midtown Club Discord server, and it still crashes upon finishing a race and clicking on "Continue" even with that MM2Hook version he provided. According to MysteryDriver, it was because apparently the race was taking place from day to evening or vice versa and too many lights.
